modules/org.restlet/src/test/java/org/restlet/engine/util/Base64Test.java

```diff
@@ -0,0 +1,65 @@
+package org.restlet.engine.util;
+
+import junit.framework.TestCase;
+
+public class Base64Test extends TestCase {
+
+    public void test_restlet_base64_encoding_without_newlines_should_be_equal_jdk_base64_encoding() {
+        String source =
+                "Man is distinguished, not only by his reason, but by this singular passion from " +
+                        "other animals, which is a lust of the mind, that by a perseverance of delight " +
+                        "in the continued and indefatigable generation of knowledge, exceeds the short " +
+                        "vehemence of any carnal pleasure.";
```

Comment:
Extract to a variable ? Seems to be used multiple times.

Reply:
thanks @quilicicf